bessctl/bessctl

```diff
@@ -161,8 +131,7 @@ def run_cli():
         print >> stderr, 'Perhaps bessd daemon is not running locally? ' \
```

Comment:
Maybe I'm doing something wrong, I get an error about stderr not being defined here